### PR TITLE
Fix small bugs / Small architecture changes

### LINKED ISSUE #88: What What?
On event:
- [x] should we set self_submit_* to false by default on the schema
- [x] should we be setting semester in an after create hook?

On attendance:
- [x] should we set present and excused to false by default on the schema

### LINKED ISSUE #96: Fix Outstanding Small Issues
- [x] Improve event listing UI
- [x] Improve update user status UI
  - [x] Add select2 to multi-select boxes
- [x] chairmen events adding

### PR DESCRIPTION
**Pull request by valleyjo**
Close #88. Close #96.

### CODE REVIEW

File: app/controllers/attendance/fines_controller.rb [hunk 1]
```diff
@@ -19,9 +19,14 @@ def update
...
   def update_fines
+    created = false
     unfined = Attendance.find_unfined
-    unfined.each { |a| Fine.create(attendance: a) }
-    head :ok
+    unfined.each { |a| created = Fine.create(attendance: a) }
+    if created
+      head :created
+    else
+      head :ok
```
Comment (D-Land) [inline app/controllers/attendance/fines_controller.rb]: This seems like it should throw up an alert or something?

File: app/models/event.rb [hunk 2]
```diff
@@ -28,18 +32,32 @@ class Event < ActiveRecord::Base
   # Helpers ####################################################################
 
   def attended_users
-    self.attendances.where(present: true).collect(&:user)
+    self.attendances.where(present: true).collect(&:user).sort_by {
```
Comment (D-Land) [inline app/models/event.rb]: Fix these ^

File: app/models/event.rb [hunk 3]
```diff
@@ -28,18 +32,32 @@ class Event < ActiveRecord::Base
...
+      |u| [u.last_name, u.first_name]
```
Comment (D-Land) [inline app/models/event.rb]: ^ and this

File: app/models/event.rb [hunk 4]
```diff
@@ -12,11 +13,14 @@ class Event < ActiveRecord::Base
...
   before_save { self.name = name.titleize }
+  before_validation { self.semester = current_semester(Time.now.year) }
 
   # Scopes #####################################################################
 
-  scope :excusable, -> { where("self_submit_excuse = :value AND date > :today",
-                               today: Date.today, value: true) }
+  scope :excusable, -> {
+    where("self_submit_excuse = :value AND date > :today", today: Date.today,
+          value: true)
```
Comment (D-Land) [inline app/models/event.rb]: Is this just formatting?

**Comment by D-Land** [comment]
Fix the hound errors about using map and not having blocks start on same line are legit things you should fix.

Also made a comment on fines controller.

File: app/models/event.rb (hunk 4 above)
Comment (valleyjo) [inline app/models/event.rb]: yea

**Pull request merged**